### PR TITLE
Addressing PR comments

### PR DESCRIPTION
**Pull request by issackjohn**

### CODE REVIEW

**Comment by issackjohn** [comment]
@lpardosixtosMs What do you think about putting "expandableItem" "nonExpandableItem" "list" In an enum?

File: resources/todomvc/big-dom-generator/src/tree-generator.js [hunk 1]
```diff
@@ -8,26 +8,49 @@ const random = new LCG(DEFAULT_SEED_FOR_RANDOM_NUMBER_GENERATOR);
...
+            currentNode.children.push({ type: childType, children: [] });
             currentNode = currentNode.children[currentNode.children.length - 1];
             currentDepth++;
-            totalNodes += nodeWeight[type];
+            totalNodes += nodeWeight[childType];
         }
-        if (MAX_GENERATED_DOM_DEPTH % 2) {
-            currentNode.type = "closedItem";
-            totalNodes = totalNodes - nodeWeight["openItem"] + nodeWeight["closedItem"];
+        // So these branches actually end in a todo list (nonExpandableItem) instead of a sprint folder.
```
Comment (lpardosixtosMs) [inline resources/todomvc/big-dom-generator/src/tree-generator.js]: NIT: reword "Ensure the last node is not an expandableItem"

File: resources/todomvc/big-dom-generator/params.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ export const DEFAULT_SEED_FOR_RANDOM_NUMBER_GENERATOR = 4212021;
 export const MAX_GENERATED_DOM_DEPTH = 40;
 export const MAX_NUMBER_OF_CHILDREN = 16;
 export const PROBABILITY_OF_HAVING_CHILDREN = 0.3;
+// The target node count for the generated DOM tree.
```
Comment (lpardosixtosMs) [inline resources/todomvc/big-dom-generator/params.js]: node -> element

File: resources/todomvc/big-dom-generator/params.js [hunk 3]
```diff
@@ -4,6 +4,9 @@ export const DEFAULT_SEED_FOR_RANDOM_NUMBER_GENERATOR = 4212021;
...
 export const TARGET_SIZE = 6000;
+// This constant is used to set the minimum number of branches in
```
Comment (lpardosixtosMs) [inline resources/todomvc/big-dom-generator/params.js]: Rewrite: The generator will ensure that the generated DOM tree has at least MIN_NUMBER_OF_MAX_DEPTH_BRANCHES of depth MAX_GENERATED_DOM_DEPTH.

**Review by lpardosixtosMs — APPROVED** [review]
LGTM with some comments.

File: resources/todomvc/big-dom-generator/params.js (hunk 3 above)
Comment (issackjohn) [reply resources/todomvc/big-dom-generator/params.js]: Done.

File: resources/todomvc/big-dom-generator/params.js (hunk 2 above)
Comment (issackjohn) [reply resources/todomvc/big-dom-generator/params.js]: Done.

File: resources/todomvc/big-dom-generator/src/tree-generator.js (hunk 1 above)
Comment (issackjohn) [reply resources/todomvc/big-dom-generator/src/tree-generator.js]: Done.

**Pull request merged**